.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,53 @@
+name: auto-merge
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+jobs:
+  auto-approve:
+    name: auto-approve prs by boldlinksig
+    runs-on: ubuntu-latest
+    steps:
+      - name: auto-approve
+        uses: hmarr/auto-approve-action@v3
+        if: github.actor == 'boldlinksig'
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}" 
+      - name: Slack Notification failure
```

Comment:
I think you only need a one slack notification job in the end not repeating it on both jobs